include/ginkgo/core/solver/solver_base.hpp

```diff
@@ -590,6 +590,14 @@ class EnablePreconditionedIterativeSolver
 };
 
 
+/**
```

Comment:
Probably not the correct file for this. We could merge upper_trs.hpp and lower_trs.hpp though, leaving those two files to import a triangular.hpp header, since they are so closely related.